### PR TITLE
Add vue-tippy6 for more consistent tooltip behavior

### PR DESCRIPTION
**Pull request by samuveth**
Still need to check how much bundle size this adds.

### CODE REVIEW

**Comment by samuveth** [comment]
[Improve progress bars and tooltips](https://trello.com/c/YCPycf57/308-improve-progress-bars-and-tooltips)

**Comment by samuveth** [comment]
[Improve progress bars and tooltips](https://trello.com/c/YCPycf57/308-improve-progress-bars-and-tooltips)

**Comment by samuveth** [comment]
@bonustrack I've fixed the duplicate package issue, it didn't seem to reduce the size though.

**Comment by bonustrack** [comment]
@samuveth Does it help to use this https://vitejs.dev/config/#resolve-dedupe ?

**Comment by bonustrack** [comment]
Also the tooltip appear to quickly, there should be some minimum time on mouse over element before seeing the tooltip.

**Comment by samuveth** [comment]
> @samuveth Does it help to use this https://vitejs.dev/config/#resolve-dedupe ?

Didn't seem to change anything.

Changed the delay. We can do pretty much anything we like with vue-tippy, we can also use props like `allowHTML` and `interactive` to something like a "Learn more" button inside the tooltip that the use can click on.

**Comment by samuveth** [comment]
@bonustrack I couldn't find a way to decrease bundle size. But I think for the amount of goodies we're getting from it, it's seems worth it.

**Comment by samuveth** [comment]
@bonustrack should I resolve conflicts or how do we continue with this PR?

**Comment by bonustrack** [comment]
Resolve conflict yes @samuveth

**Comment by samuveth** [comment]
Done

**Comment by bonustrack** [comment]
Can you use the skin colors on the tooltip?

File: src/main.ts [hunk 1]
```diff
@@ -22,6 +23,13 @@ const app = createApp({
 })
   .use(i18n)
   .use(router)
+  .use(LockPlugin, options);
+app
```
Comment (bonustrack) [inline src/main.ts:27]:
```suggestion
```

**Review by bonustrack — APPROVED** [review]
Feel free to merge when change is done

**Comment by samuveth** [comment]
Couldn't find a fix for the colors. I tried with the current tooltip theme and it works well across all our space skins, also improved readability on some space like balancer where it was pretty bad before.
image

**Pull request merged**